### PR TITLE
Allow changing Schedulers RateLimitter setting during startup.

### PR DESCRIPTION
**Pull request by gmarek**
This will be needed for component performance testing.
@wojtek-t @fgrzadkowski

### CODE REVIEW

**Comment by derekwaynecarr** [comment]
@timstclair - is this what you were looking for that we discussed at OSCON?

File: plugin/pkg/scheduler/factory/factory.go [hunk 1]
```diff
@@ -285,6 +285,10 @@ func (factory *ConfigFactory) makeDefaultErrorFunc(backoff *podBackoff, podQueue
 	}
 }
 
+func (factory *ConfigFactory) SetBindPodsRateLimiter(rateLimiter util.RateLimiter) {
```
Comment (wojtek-t) [inline plugin/pkg/scheduler/factory/factory.go]: In line 43 there are two constants defined: BindPodsQps and BindPodsBurst.
They seem to be unneded now (i.e. these are default values for flags now). We should get rid of them and either just remove setting rate limitter in NewConfigFactory or pass the values of the flag there.

Comment (gmarek) [inline plugin/pkg/scheduler/factory/factory.go]: Done, PTAL

File: cmd/integration/integration.go [hunk 2]
```diff
@@ -176,7 +181,7 @@ func startComponents(firstManifestURL, secondManifestURL, apiVersion string) (st
 	handler.delegate = m.Handler
 
 	// Scheduler
-	schedulerConfigFactory := factory.NewConfigFactory(cl)
+	schedulerConfigFactory := factory.NewConfigFactory(cl, util.NewTokenBucketRateLimiter(bindPodsQps, bindPodsBurst))
```
Comment (wojtek-t) [inline cmd/integration/integration.go]: Do we really need to rate-limit it in integration tests? Can't we just pass null here?

Comment (gmarek) [inline cmd/integration/integration.go]: Done. PTAL

**Comment by wojtek-t** [comment]
LGTM

**Comment by timothysc** [comment]
@derekwaynecarr there are two of us, mine is @timothysc ;-)

**Comment by timothysc** [comment]
LGTM, but fwiw this originally used to be enabled through the command line and someone disabled.

**Comment by alex-mohr** [comment]
@k8s-bot test this [contrib/submit-queue: candidate for merging]

**Comment by alex-mohr** [comment]
Automatic merge from SubmitQueue

**Pull request merged**